src/etl/ClinicalTrialModelFactory.test.ts

```diff
@@ -0,0 +1,586 @@
+import { ClinicalTrialModelFactory } from './ClinicalTrialModelFactory'
+import { ClinicalTrialModel } from '../shared/models/ClinicalTrialModel'
+import { riphCtisDto, riphDmDto, riphJardeDto1 } from '../shared/test/helpers/elasticsearchHelper'
+
+describe('clinical trial model factory', () => {
+  const unavailable = 'INDISPONIBLE'
+
+  describe('ctis', () => {
+    it('when RIPH CTIS with all fields filled is given, should build a CTIS clinical trial model', () => {
```

Comment:
peaufinage (non bloquant): Je propose cette suggestion pour voir l'intention en premier.
```suggestion
    it('should build a CTIS clinical trial model when RIPH CTIS with all fields filled is given', () => {
```